en/admin/migration.inc

```diff
@@ -166,6 +166,14 @@ In general, the **cubrid loaddb** utility uses files (schema definition, object
 *   Trigger file(*database-name*\ **_trigger**): A file created by an unload operation. It contains information on the triggers defined in the database.
 *   User-defined object file(*user_defined_object_file*): A file in table format written by the user to enter mass data. (For details, see :ref:`howtowrite-loadfile` .)
 
+**Loading modes**
+
+The **cubrid loaddb** utility provides two modes of loading the data into a database.
+
+*   SA_MODE, which loads the data without running the server process.
```

Comment:
```suggestion
*   Stand-alone mode loads the data offline, within a single thread. CUBRID 10.1 and older versions run only in this mode.
```

SA_MODE/CS_MODE are not clear enough for user. "without running server" is also confusing; server is run in stand-alone mode :).